src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const DAY_RENT = 40;
+  const BigDiscount = 50;
+  const SmallDiscount = 20;
```

Comment:
Similarly, `SmallDiscount` should be in all uppercase with underscores. Consider renaming it to `SHORT_TERM_DISCOUNT`.